src/main/resources/templates/archive.html

```diff
@@ -34,19 +34,27 @@ <h2><span id="user"></span>'s Surveys:</h2>
         </div>
 
         <div class="container d-flex justify-content-start align-items-center mt-5 pt-2">
-            <div class="row m-4" th:each="survey : ${user.getSurveys()}">
-                <a th:href="@{|/api/survey/${survey.getID()}/questions|}" class = "custom-card">
+            <div class="survey row m-4" th:each="survey : ${user.getSurveys()}">
                     <div class = "card">
-                        <img class="card-img-top" src="/images/survey.jpg">
+                        <a th:href="@{|/survey/${survey.getID()}/responses|}" class = "custom-card">
+                            <img class="card-img-top" src="/images/survey.jpg">
+                        </a>
                         <div class = "card-body">
-                            <h4 class="card-title text-center" th:text="${survey.getName()}"></h4>
+                            <a th:href="@{|/survey/${survey.getID()}/responses|}" class = "custom-card">
+                                <h4 class="card-title text-center" th:text="${survey.getName()}"></h4>
+                            </a>
                         </div>
-                        <button th:id="${survey.getID()}">Close Survey</button>
+                        <div th:switch="${survey.closed}">
```

Comment:
I think this would look better if you changed `div` to `th:block`. Right now the button looks weird.
![image](https://user-images.githubusercontent.com/32046329/78296358-9134ee80-74fb-11ea-9edf-85fe3ea5493e.png)

```
                        <th:block th:switch="${survey.closed}">
                                ...
                        </th:block>
```